ivle-sync.py

```diff
@@ -189,17 +213,52 @@ def sync_announcements(session):
 
 
 def get_credentials():
-    userid = credentials['USERID']
-    if userid == '':
-        userid = input("UserID: ")
+    userid = input("UserID: ")
 
-    password = credentials['PASSWORD']
-    if password == '':
-        password = getpass("Password: ")
+    password = getpass("Password: ")
 
     return (userid, password)
 
 
+def clear_token():
+    try:
+        del credentials['TOKEN']
+        write_credentials()
+        print("Token cleared.")
+    except:
+        print("No token is set.")
+        exit(-1)
+
+
+def write_credentials():
+    try:
+        with open(
+                join(dirname(realpath(__file__)), 'credentials.json'),
+                'w',
+                encoding='utf-8') as file:
+            json.dump(credentials, file)
+
+    except:
+        print("Error writing to credentials.json")
+        exit(-1)
+
+
+def yes_no():
+    yes = set(['yes', 'y'])
+    no = set(['no', 'n'])
+
+    while True:
+        choice = input(
+            "Do you want to write the token obtained to credentials.json?[y/n] "
+        ).lower()
+        if choice in yes:
+            return True
+        elif choice in no:
```

Comment:
Since this is not an irreversible action, it is okay to have a default option which should be `n`